vss-processor-plugin/buildSrc/src/main/kotlin/version.gradle.kts

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Refactored the version handling so that they are not stored inside the extras anymore. This means the redundant Version plugin could be removed. However reading the version.txt file still requires the SemanticVersion class in both places.